ui/keyboardMove/src/plugins/keyboardMove.ts

```diff
@@ -31,6 +32,9 @@ export default (opts: Opts) => {
     if (!submitOpts.isTrusted) return;
     // consider 0's as O's for castling
     v = v.replace(/0/g, 'O');
+    if (v.match(iccfRegex)) {
```

Comment:
Keep in mind that this function is run on every keypress, so this regex will never be matched until the entire ICCF notation is entered. You essentially want to allow partial ICCF inputs somehow.